packages/@uppy/provider-views/src/style/uppy-ProviderBrowser-viewType--grid.scss

```diff
@@ -115,3 +115,8 @@
     opacity: 1;
   }
 }
+
+.grid-checkbox:focus + label {
```

Comment:
We need a “safe” prefixed name for classes, how about `uppy-ProviderBrowserItem-checkbox--grid` or similar?